contracts/AxelarGateway.sol

```diff
@@ -327,18 +327,23 @@ contract AxelarGateway is IAxelarGateway, AdminMultisigBase {
 
             // Prevent a re-entrancy from executing this command before it can be marked as successful.
             _setCommandExecuted(commandId, true);
+            // solhint-disable-next-line avoid-low-level-calls
             (bool success, ) = address(this).call(abi.encodeWithSelector(commandSelector, params[i], commandId));
 
-            if (success) emit Executed(commandId);
-            else _setCommandExecuted(commandId, false);
+            if (success) {
+                emit Executed(commandId);
+
+                // Not current operators anymore
+                if (commandSelector == AxelarGateway.transferOperatorship.selector) currentOperators = false;
+            } else _setCommandExecuted(commandId, false);
         }
     }
 
     /******************\
     |* Self Functions *|
     \******************/
 
-    function deployToken(bytes calldata params, bytes32) external onlySelf {
+    function deployToken(bytes calldata params, bytes32) external payable onlySelf {
```

Comment:
Why payable

Reply:
Again, saves some gas, again, don't think it matters enough.